crates/sim2h/src/cache.rs

```diff
@@ -49,7 +47,7 @@ impl Space {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
```

Comment:
This change is worth reviewing, I believe it to be correct because you can construct one of these from an empty `HashMap` anyway, therefore there must not be an invariant violated by this `Default` `impl`. It might not be the API we want though.